### PR TITLE
Validate .cirrus.star when running cirrus validate

### LINKED ISSUE #145: Validate .cirrus.star when running cirrus validate
Currently only YAML configuration is validated.

Logic from `cirrus run` can probably be re-used for this one.

### PR DESCRIPTION
**Pull request by edigaryev**
Adds `-e` and `--experimental-parser` arguments from `cirrus run`.

The `cirrus run`'s `-f` argument now can be used to only validate a specific configuration, either one ending with `.yml` or `.star` for YAML and Starlark correspondingly.

Resolves #145.

### CODE REVIEW

File: internal/commands/run.go [hunk 1]
```diff
@@ -83,19 +83,33 @@ func readYAMLConfig() (string, error) {
 	return string(yamlConfig), nil
 }
 
-func readStarlarkConfig(ctx context.Context, env map[string]string) (string, error) {
-	starlarkSource, err := ioutil.ReadFile(".cirrus.star")
+func readStarlarkConfig(ctx context.Context, path string, env map[string]string) (string, error) {
```
Comment (fkorotkov) [inline internal/commands/run.go:86]: Maybe rename it to `evaluateStarlarkConfig` or `readAndEvaluateStarlarkConfig` since it translating Starlark to YAML?

**Review by fkorotkov — APPROVED** [review]
Left a small rename consideration

File: internal/commands/run.go (hunk 1 above)
Comment (edigaryev) [reply internal/commands/run.go]: Renamed in https://github.com/cirruslabs/cirrus-cli/pull/169/commits/fd2fe7e15488ce783b641f70fd660357b90f0893 to `EvaluateStarlarkConfig`.

**Review by fkorotkov — APPROVED** [review]

**Comment by edigaryev** [comment]
Funny, but seems that we'll either won't support `cirrus validate` (1) or validating Starlark configurations (2) on Solaris for now due to https://github.com/google/starlark-go/issues/320.

@fkorotkov which option do you prefer?

**Comment by edigaryev** [comment]
Ironic, but seems that we'll either won't support cirrus validate (1) or validating Starlark configurations (2) on Solaris for now due to google/starlark-go#320.

@fkorotkov which option do you prefer?

**Comment by fkorotkov** [comment]
Let's see what Startlark folks will answer. 🤔

**Pull request merged**